airbyte-webapp-e2e-tests/cypress/pages/replicationPage.ts

```diff
@@ -9,6 +9,9 @@ const destinationNamespaceSource = "div[data-testid='namespaceDefinition-source'
 const destinationNamespaceCustomInput = "input[data-testid='input']";
 const syncModeDropdown = "div[data-testid='syncSettingsDropdown'] input";
 const cursorFieldDropdown = "button[class^='PathPopoutButton_button']";
+const streamFieldNames = "[class^='TreeRowWrapper_rowWrapper'] span";
+const streamDataTypes = "[class^='TreeRowWrapper_rowWrapper'] div:nth-child(2)";
+const arrowStream = "[class^='Arrow_container__']";
```

Comment:
what is an arrow stream? 🙂

Reply:
<img width="96" alt="Screenshot 2022-12-06 at 21 59 24" src="https://user-images.githubusercontent.com/108751383/206010492-ffa7a349-3f5f-4228-ac72-6b98870abfeb.png">
button for open stream details